ios/FasterImageViewManager.swift

```diff
@@ -96,7 +97,11 @@ final class FasterImageView: UIView {
                 }
                 progressiveLoadingEnabled = options.progressiveLoadingEnabled ?? false
                 grayscale = options.grayscale ?? 0.0
-                url = options.url
+
+                var _urlRequest = URLRequest(url: URL(string: options.url)!)
+                _urlRequest.allHTTPHeaderFields = options.headers
+
+                urlRequest = _urlRequest
```

Comment:
```suggestion
               if let url = URL(string: options.url) {
                var urlRequestFromOptions = URLRequest(url: url)
                urlRequestFromOptions.allHTTPHeaderFields = options.headers

                urlRequest = urlRequestFromOptions
                } else {
                onError?([
                    "error": "Expected a valid url but got: \(url ?? "nil")",
                ])
                }
```

Reply:
I'd rather show an error here if the URL is missing rather than crash; most folks don't run their RN projects in Xcode when developing, so it would at least give them an error in the console.

Reply:
Thanks! I applied your feedback.